frontend/src/lib/router.tsx

```diff
@@ -430,6 +430,7 @@ export const ROUTES: {
 export const NotFoundRoute = {
   path: '*',
   exact: true,
+  name: `Whoops! This page doesn't exist`,
```

Comment:
This string is translated because it's used on the 404 page.

I didn't look through every single title. Checked many and they seemed already translated (I guess because the side bar ones are translated already).

Reply:
This is translated in the frontend/src/components/App/RouteSwitcher.tsx PageTitle component when the route is switched.